src/attach.rs

```diff
@@ -11,14 +12,18 @@ use crate::kvm;
 pub fn attach(opts: &InspectOptions) -> Result<()> {
     println!("attaching");
 
-    let vm = try_with!(
+    let vm = Arc::new(try_with!(
         kvm::hypervisor::get_hypervisor(opts.pid),
         "cannot get vms for process {}",
         opts.pid
-    );
+    ));
+    vm.stop()?;
 
-    let device = try_with!(Device::new(&Arc::new(vm)), "cannot create vm");
+    let device = try_with!(Device::new(&vm.clone()), "cannot create vm");
+    vm.resume()?;
     device.create();
     device.create();
+    println!("pause");
+    std::thread::sleep(Duration::from_secs(99999));
```

Comment:
Maybe use the actual pause syscall?

```suggestion
    unistd::pause();
```